mewarpx/mewarpx/mcc_wrapper.py

```diff
@@ -71,6 +107,8 @@ def __init__(self, electron_species, ion_species, P_INERT,
             elif file_name in ion_collision_types:
                 scatter_dict = {"cross_section": path}
                 ion_scattering_processes[ion_collision_types[file_name]] = scatter_dict
+            else:
+                raise ValueError(f"{path}: file not recognized")
```

Comment:
```suggestion
                raise ValueError(
                    f"{path}: filename not recognized as an MCC cross-section "
                    "file. Please move outside this folder or end with "
                    "something other than .dat if it is not a cross-section "
                    "file."
                )
```

Reply:
Also I somehow completely missed f-strings, awesome! I think this is the first one in the code :)